core/src/main/scala/org/apache/spark/deploy/master/ui/ApplicationPage.scala

```diff
@@ -70,6 +70,16 @@ private[ui] class ApplicationPage(parent: MasterWebUI) extends WebUIPage("app")
             }
             </li>
             <li>
+              <span data-toggle="tooltip" title={ToolTips.APPLICATION_EXECUTOR_LIMIT}
+                    data-placement="right">
+                <strong>Executor Limit: </strong>
+                {
+                  if (app.executorLimit == Int.MaxValue) "Unlimited" else app.executorLimit
```

Comment:
I noticed that `ApplicationDescription` has a `maxCores` field which is displayed on the Master UI's applications page. Do you think that there's any possibility for confusion between the limits displayed there (which are measured in terms of cores) and the executor limits shown here?